migrate/20230915_postgres.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    create_table(:postgres_server) do
+      column :id, :uuid, primary_key: true, default: nil
+      column :created_at, :timestamptz, null: false, default: Sequel.lit("now()")
+      column :updated_at, :timestamptz, null: false, default: Sequel.lit("now()")
+      column :project_id, :project, type: :uuid, null: false
+      column :location, :text, collate: '"C"', null: false
+      column :server_name, :text, collate: '"C"', null: false, unique: true
+      column :target_vm_size, :text, collate: '"C"', null: false
+      column :target_storage_size_gib, :bigint, null: false
+      column :superuser_password, :text, collate: '"C"', null: false # TODO encrypt this
+      column :vm_id, :vm, type: :uuid
+    end
+
+    alter_table(:billing_record) do
+      drop_constraint(:billing_record_resource_id_span_excl)
+      add_exclusion_constraint([[:resource_id, "="], [:billing_rate_id, "="], [:span, "&&"]])
+    end
```

Comment:
This should be good to chip out into another patch, which would probably simplify this patch to a `change` block and get the new constraint tested, since it seems like this is more a change to how billing constraints work in general...and perhaps an oversight in the original?

Reply:
It was oversight. I thought about this case, but forgot about it during implementation. I extracted this to separate PR and merged it.